tests/syntax/test_operators.py

```diff
@@ -503,6 +503,105 @@ def test_object_in_region_3d():
     assert p == (True, True, False, False)
 
 
+# Intersects
+def test_intersects_obj_obj():
+    p = sampleParamPFrom(
+        """
+        obj1 = new Object at (-1,0,0.1), with allowCollisions True
+        obj2 = new Object at (1,0,0), with allowCollisions True
+        obj3 = new Object with width 10, with length 10, with height 10, with allowCollisions True
+        param p = tuple([obj1 intersects obj2, obj1 intersects obj3, obj2 intersects obj3])
```

Comment:
There's no longer any need to write `tuple([...])` rather than just `(...)` (the old tests did that because we repurposed the tuple notation for vectors prior to Scenic 2).

Reply:
Good point, I went ahead and fixed this test and all the others in the file.